browser/app.css

```diff
@@ -10,6 +10,13 @@ header {
   text-align: center;
 }
 
+.label-panel {
+  display: inline-block;
+  max-width: 100%;
+  margin-bottom: 5px;
+  width: 1439px;
```

Comment:
We do not need width of so much pixels. Instead use `width: 100%` and remove `max-width` parameter.

Why are we using `margin-bottom` here. It will not add anything as it is already inside div container. So we can get rid of this also.